emerge-gitclone

```diff
@@ -18,13 +18,20 @@ with open('/usr/share/flatcar/release') as release_file:
 commits = {}
 if release:
 	manifest = "https://raw.githubusercontent.com/kinvolk/manifest/v%s/release.xml" % release
+	if "+" in release:
+		dev_build_tag = release.split("+")[1]
+		manifest = "https://raw.githubusercontent.com/kinvolk/manifest-builds/%s/%s.xml" % (dev_build_tag, dev_build_tag)
+	else:
+		manifest = "https://raw.githubusercontent.com/kinvolk/manifest/v%s/release.xml" % release
 	try:
 		from xml.dom.minidom import parseString as pxs
 		from urllib import request as req
 		for repo in pxs(req.urlopen(manifest).read()).getElementsByTagName('project'):
 			commits[repo.getAttribute('name')] = repo.getAttribute('revision')
 	except Exception:
 		print(">>> Failed to read manifest commits for %s" % release)
+		sys.exit(1)
+else:
```

Comment:
```suggestion

```
I think the else case here is not needed anymore then.

Not sure when we would hit it, is it in local building? Would be good to know which code path the version of the SDK container dev builds take now.

Reply:
Done